testing/acceptance_test.py

```diff
@@ -541,6 +542,28 @@ def test_foo(data):
         res = pytester.runpytest(p)
         res.assert_outcomes(passed=3)
 
+    @pytest.mark.filterwarnings("ignore:'encoding' argument not specified")
```

Comment:
```suggestion
    # Warning ignore because of:
    # https://github.com/python/cpython/issues/85308
    # Can be removed once Python<3.12 support is dropped.
    @pytest.mark.filterwarnings("ignore:'encoding' argument not specified")
```